File: Taller java/src/nivel_2/Calcular_Media.java
package nivel_2;
import java.util.Scanner;
public class Calcular_Media {
public static void calcularMedia() {
        Scanner sc = new Scanner(System.in);

        System.out.print("Ingrese varios números separados por comas: ");
        String entrada = sc.nextLine();

        String[] partes = entrada.split(",");
        double suma = 0;
        for (String num : partes) {
            suma += Double.parseDouble(num.trim());
        }

        double media = suma / partes.length;
        System.out.println("La media de los números es: " + media);
    }

    public static void main(String[] args) {
        calcularMedia();
    }
}

File: programacion niveles Python/Nivel 2/Calcular_Media.py
def calcular_media(lista):
    numeros = input(" Ingrese varios numeros separados por comas:")
    lista = [float(x.strip() for x in numeros.split(",")]
    if len (lista) == 0:
        print("No ingresaste ningun numero valido")
    else: 
    media = sum(lista) / len(lista)
    print("La media de los numeros es: {media}")

# llamar la función 
calcular_media()

